python-package/lightgbm/sklearn.py

```diff
@@ -316,7 +316,7 @@ def fit(self, X, y,
         group : array-like or None, optional (default=None)
             Group data of training data.
         eval_set : list or None, optional (default=None)
-            A list of (X, y) tuple pairs to use as a validation sets.
+            A list of (X, y) tuple pairs to use as a validation set.
```

Comment:
LightGBM supports multiple validation sets, so please leave `sets`.

Reply:
Fixed it.

Reply:
Thanks again! Will merge right after passing tests.